src/EnergyPlus/HVACVariableRefrigerantFlow.hh

```diff
@@ -773,7 +775,7 @@ namespace HVACVariableRefrigerantFlow {
               MySuppCoilPlantScanFlag(true), airLoopNum(0), isInOASys(false), isInAirLoop(false), isInZone(false), isSetPointControlled(false),
               coolSPActive(false), heatSPActive(false), coolLoadToSP(0.0), heatLoadToSP(0.0), coilTempSetPoint(0.0), suppTempSetPoint(0.0),
               controlZoneMassFlowFrac(1.0), zoneSequenceCoolingNum(0), zoneSequenceHeatingNum(0), coolCoilAirInNode(0), coolCoilAirOutNode(0),
-              heatCoilAirInNode(0), heatCoilAirOutNode(0)
+              heatCoilAirInNode(0), heatCoilAirOutNode(0), coolingCoilActive(false)
         {
```

Comment:
May want to take this opportunity to initialize all of these fields in the declaration and eliminate the default constructor. Or not.

Reply:
@amirroth Thanks. Your suggestion will be addressed once the when the functional changes are accepted by a reviewer.

Reply:
Done.